README.rst

```diff
@@ -53,6 +53,11 @@ You can install from source by cloning this repository and issuing a pip install
     python setup.py sdist
     pip install dist/sagemaker-1.7.1.tar.gz
 
+Supported Operating Systems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+SageMaker Python SDK supports Unix/Linux and Mac. Support for windows is not available now.
```

Comment:
capitalize "Windows". might want to specify that Windows 10 does work as it can run bash commands

Reply:
Just removed mention of 'Windows'. I feel weird because most docs I have ever read only say what platforms being supported.